### PR TITLE
Add `make` to list of Fedora dependencies

### PR DESCRIPTION
**Pull request by WillNilges**
I needed to install it before building Xournalpp

### CODE REVIEW

**Comment by Technius** [comment]
The installation instructions in the readme don't assume any specific build tool (other than CMake), but they do require `make`, `ninja`, or another supported CMake generator. I think it's better to have a small note indicating that such a tool must be installed instead of specifying `make` as a hard dependency.

**Comment by WillNilges** [comment]
> The installation instructions in the readme don't assume any specific build tool (other than CMake), but they do require `make`, `ninja`, or another supported CMake generator. I think it's better to have a small note indicating that such a tool must be installed instead of specifying `make` as a hard dependency.

That'd probably be better tbh.

File: readme/LinuxBuild.md [hunk 1]
```diff
@@ -6,6 +6,8 @@ Please create pull requests (or file issues) if you have more precise dependenci
 
 Lua is needed for plugins, if it is missing, the plugins will be disabled.
 
+The installation instructions don't assume any specific build tool (other than CMake), but they do require make, ninja, or another supported CMake generator. It is required that such a tool is installed in order to build xournalpp.
```
Comment (Technius) [inline readme/LinuxBuild.md]: Very minor nitpick, but could you break this line into multiple lines with less than 80 (or 90) characters to be consistent with the rest of the file?

**Comment by Technius** [comment]
LGTM besides the minor nitpick (see above).

**Comment by LittleHuba** [comment]
Add notice that a tool like make is required for manual build to instructions

**Pull request merged**